cirq-core/cirq/transformers/gauge_compiling/gauge_compiling.py

```diff
@@ -154,12 +151,16 @@ def __call__(self, prng: np.random.Generator) -> Gauge:
 
 @transformer_api.transformer
 class GaugeTransformer:
+
     def __init__(
         self,
         # target can be either a specific gate, gatefamily or gateset
         # which allows matching parametric gates.
         target: Union[ops.Gate, ops.Gateset, ops.GateFamily],
         gauge_selector: Callable[[np.random.Generator], Gauge],
+        symbolize_2_qubit_gate_fn: Optional[
+            Callable[[ConstantGauge, sympy.Symbol], Tuple[ops.Gate, float]]
```

Comment:
lets allow for some flexibility and make the signature deal with lists

```py3
Callable[[ConstantGauge, Sequence[sympy.Symbol]], Sequence[Tuple[ops.Gate, Union[float, int]]]]
```

Reply:
Good point. Done, but not exactly used the type you mentioned here.